### ISSUE TITLE
Kez Raptor Dance, Doesn't dispel the HEX effect upon completion

### ISSUE THREAD

**Issue by Jolyrogue** [ValveSoftware/Dota2-Gameplay#23077]
### Ability name

Raptor dance

### Description

Kez Raptor Dance, Doesn't dispel the HEX effect upon completion, unlike bladefury Juggernaut, in the description of which is written the same as Kez

### Example Match ID (and possibly Timestamp)

8058464139 - 20:58

**Comment by eva78mason** [comment]
Thanks for sharing this! It seems like there's a discrepancy in the description of Kez's Raptor Dance ability compared to Bladefury Juggernaut. While both descriptions mention not dispelling the HEX effect upon completion, it might be a bug or an oversight in the game.

**Comment by Jolyrogue** [comment]
no, the bug is that both abilities use strong dispel, and it in turn should dispel the effect of HEX, in the case of juggernaut it works correctly, but in the case of Kez it doesn't
